podcasts/Kids Profile/KidsProfileBannerView.swift

```diff
@@ -0,0 +1,108 @@
+import SwiftUI
+
+struct KidsProfileBannerView: View {
+    @EnvironmentObject var theme: Theme
+
+    let viewModel: KidsProfileBannerViewModel
+
+    var body: some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: 8)
+                .fill(theme.primaryUi01Active)
+
+            HStack {
+                textsView
+                    .padding(.leading, 16)
+                    .padding(.trailing, 35)
+
+                Spacer()
+
+                ZStack(alignment: .topTrailing) {
+                    VStack {
+                        Spacer()
+
+                        Image("kids-profile-banner-face")
+                            .clipShape(RoundedRectangle(cornerRadius: 8))
+                    }
+
+                    VStack {
+                        Button(action: viewModel.closeButtonTap) {
+                            Image(systemName: "xmark")
+                                .imageScale(.small)
+                                .foregroundColor(theme.secondaryIcon02)
+                        }
+                        .frame(width: 24, height: 24)
+                        .padding(.top, 8)
+                        .padding(.trailing, 4)
+
+                        Spacer()
+                    }
+                }
+            }
+        }
+        .frame(height: 105)
+    }
+
+    var textsView: some View {
+        VStack(alignment: .leading) {
+            HStack {
+                Text(L10n.kidsProfileBannerTitle)
+                    .font(
+                        Font.custom("SF Pro Text", size: 15.0)
```

Comment:
Will this adapt if the user sets a larger font on Accessibility settings?

Reply:
Yep!